File: Searching/Linear_Search.cs

namespace DSA_Programs.Searching
{
    public class Linear_Search
    {

        public static void linearSearch(int[] array, int target)
        {
            int size = array.Length;
            for (int i = 0; i < size; i++)
            {
                if (array[i] == target)
                {
                    Console.WriteLine($"Element Found at index : {i}");
                    break;
                }
            }
        }

        public static void Run()
        {
            Console.Write("Enter the size of Array : ");
            int size = int.Parse(Console.ReadLine());
            int[] array = new int[size];

            Console.WriteLine("Enter the elements in Array :");
            for (int i = 0; i < size; i++)
            {
                array[i] = int.Parse(Console.ReadLine());
            }

            Console.WriteLine();
            Console.Write("Enter the target element : ");
            int target = int.Parse(Console.ReadLine());

            linearSearch(array, target);
        }
    }
}

File: Algorithms/Searching/Linear_Search.js
const array = [8, 4, 5, 2, 7, 9, 3, 1];
const target = 9;


function Linear_Search(array, target) {
    const size = array.length;
        for (let i = 0; i < size; i++) {
            if (array[i] === target) {
                return i;
            }
        }
        return -1;
}

const index = Linear_Search(array, target);
if(index !== -1) console.log("Element Found at Index: ",index);
else console.log("Element Not Found !!");